common/oauthTokenManager.go

```diff
@@ -731,6 +731,9 @@ func jsonToTokenInfo(b []byte) (*OAuthTokenInfo, error) {
 	if err := json.Unmarshal(b, &OAuthTokenInfo); err != nil {
 		return nil, err
 	}
+	if OAuthTokenInfo.TokenRefreshSource == TokenRefreshSourceTokenStore {
```

Comment:
Is there a good way to UT this? We can try to add a UT or add this to the Jan backlog to test the env var

Reply:
Would this be possible to test in the python tests we have?

Reply:
Does not really, we need a way to put oauth token in the very specific path like Stg exp, and better way may be to move to a more away from this to a more testable way than complicate the existing testcases.